clloader/scenarios/instance_incremental.py

```diff
@@ -27,10 +28,11 @@ class InstanceIncremental(_BaseCLLoader):
     def __init__(
         self,
         cl_dataset: _ContinuumDataset,
-        nb_tasks: int,
+        nb_tasks: int = 0,
```

Comment:
Ca veut dire quoi en instance incremental nb_tasks=0? pour moi c'est un argument obligatoire dans ce scenario

Reply:
If we want to use the default number of tasks. Like for CORe50, there are 7 tasks by default, if the user left the `nb_tasks` unspecified (here `0`, but it could be `-1`) we have 7 tasks.

However the user could provide a number lower than 7, and we would truncate those tasks.

Note that if the desired `nb_tasks` is above the one designed by the dataset, we raise an error.

Reply:
ok